File: ReactiveServer/src/main/java/com/nospoon/vertxserver/core/messagehandlers/PlayerAttacher.java
package com.nospoon.vertxserver.core.messagehandlers;

import com.nospoon.vertxserver.core.model.Player;

import java.util.List;

/**
 * Created by Nestor on 8/11/2016.
 */
public interface PlayerAttacher {


void attachPlayers(List<Player> players);

void detachPlayers(List<Player> players);

void attachPlayer(Player player);

void detachPlayer(Player player);

}

File: ReactiveServer/src/main/java/com/nospoon/vertxserver/core/messagehandlers/PlayerAttacher.kt
package com.nospoon.vertxserver.core.messagehandlers

import com.nospoon.vertxserver.core.model.Player

/**
 * Created by Nestor on 8/11/2016.
 */
interface PlayerAttacher {


    fun attachPlayers(players: List<Player>)

    fun detachPlayers(players: List<Player>)

    fun attachPlayer(player: Player)

    fun detachPlayer(player: Player)

}
